src/Symfony/Component/Form/Extension/Validator/ValidatorTypeGuesser.php

```diff
@@ -264,7 +264,7 @@ protected function guess($class, $property, \Closure $closure, $defaultValue = n
                 $constraints = $memberMetadata->getConstraints();
 
                 foreach ($constraints as $constraint) {
-                    if ($guess = $closure($constraint)) {
+                    if ($guess = $callback($constraint)) {
```

Comment:
I'm thinking of reverting this change as well as the one in `FormTypeGuesserChain`. This method can't be used directly either, and while extending the class could allow using other callables, it's quite an edge case.